specification/redis/resource-manager/Microsoft.Cache/preview/2019-07-01/redis.json

```diff
@@ -1347,9 +1383,187 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/redis/{cacheName}/privateLinkResources": {
```

Comment:
>privateLinkResources [](start = 116, length = 20)

Modification of an existing stable API version is not allowed anymore. (See [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#versioning) for details.) You will need to follow the instructions in the list to start the breaking changes review.